docs/topics/multiplatform/multiplatform-configure-compilations.md

```diff
@@ -57,10 +56,9 @@ kotlin {
 
 ```groovy
 kotlin {
-    jvm().compilations.all {
+    tasks.withType(KotlinJvmCompile).configureEach {
```

Comment:
Why we've switched to configuration of compile tasks here? The article was about configuring compilations